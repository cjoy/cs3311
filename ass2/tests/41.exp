Allison Memari (3306815)
Suggestions for study in 11s1

Course   Title                 UOC Requirement
-------- -----                ---- -----------
COMP1917 Computing 1             6 Level 1 Core Courses
ENGG1000 Engineering Design      6 Level 1 Core Courses
MATH1081 Discrete Mathematics    6 Level 1 Core Courses
MATH1131 Mathematics 1A          6 Level 1 Core Courses
MATH1141 Higher Mathematics 1    6 Level 1 Core Courses
Free.... Free Electives (many   12 Free elect
