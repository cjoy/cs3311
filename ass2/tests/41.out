Allison Memari (3306815)
Suggestions for study in 11s1

165 167Course   Title                 UOC Requirement
-------- -----                ---- -----------
